### DISCUSSION TITLE
Missing Docker Images (version 2.2)

### DISCUSSION THREAD

**Discussion by aberchtold** [microsoft/dotnet-framework-docker#992] [Q&A] [answered]
While running an old pipeline, I noticed that the docker images were moved to the Microsoft Container Registry. However, the MCR seems to be missing certain images. 

For version 2.1 it works without issues:
`docker pull mcr.microsoft.com/dotnet/sdk:2.1`
`docker pull mcr.microsoft.com/dotnet/aspnet:2.1`

Version 2.2 is missing:
`docker pull mcr.microsoft.com/dotnet/sdk:2.2`
`docker pull mcr.microsoft.com/dotnet/aspnet:2.2`

Error messages:
`Error response from daemon: manifest for mcr.microsoft.com/dotnet/sdk:2.2 not found: manifest unknown: manifest tagged by "2.2" is not found`
`Error response from daemon: manifest for mcr.microsoft.com/dotnet/aspnet:2.2 not found: manifest unknown: manifest tagged by "2.2" is not found`

I'm aware that these versions are old and have reached end of life but since version 2.1 is still available, I was curious about why 2.2 was removed.

Is there some way to find the 2.2 images?

**Comment by mthalman** [comment, marked as answer]
This repo is specific to .NET Framework. Since you're asking about .NET Core, it's more relevant in the https://github.com/dotnet/dotnet-docker repo. However, I can still answer this here.

Back when .NET Core 2.1 and 2.2 were released the Docker repository names being used were `dotnet/core/sdk` and `dotnet/core/aspnet`. As part of the release of .NET 5, there was a [migration to different repository names](https://github.com/dotnet/dotnet-docker/discussions/3667): `dotnet/sdk` and `dotnet/aspnet`. That migration occurred after .NET Core 2.2 was EOL, so 2.2 tags were not included in that migration. But 2.1 was still in support so those tags were included in both locations to avoid a breaking change.

So the short answer is that you can find 2.2 tags at `mcr.microsoft.com/dotnet/core/sdk:2.2` and `mcr.microsoft.com/dotnet/core/aspnet:2.2`.

**Reply by aberchtold** [reply]
Sorry about using the wrong repo.

But thanks a lot for the quick and clear answer. Appreciate it!
